hil/cli/port.py

```diff
@@ -11,9 +13,29 @@ def port():
 @port.command(name='show')
 @click.argument('switch')
 @click.argument('port')
-def port_show(switch, port):
+@click.option('--jsonout', is_flag=True)
+def port_show(switch, port, jsonout):
```

Comment:
After populating the db with `examples/dbinit.py`, if I do a `port show mock01 gi1/0/1` I get the error `list index out of range`.  It doesn't work on the other ones either. However, it does work if I add new ports from the CLI rather than the script, which is odd.  This same scenario does work in `master`, which leads me to think something here is breaking `show port`.